deployment/constructor_params/lynx/child.yml

```diff
@@ -3,27 +3,28 @@ deployment:
   chain_id: 80001
 
 artifacts:
-    dir: ./deployment/artifacts/
-    filename: lynx.json
+  dir: ./deployment/artifacts/
+  filename: lynx.json
 
 contracts:
-    - MockPolygonChild
-    - ProxyAdmin
-    - LynxTACoChildApplication:
-        _rootApplication: $MockPolygonChild
-    - TransparentUpgradeableProxy:
-        _logic: $LynxTACoChildApplication
-        admin_: $ProxyAdmin
-        _data: $bytes:0x
-    - LynxRitualToken:
-        _totalSupplyOfTokens: 10000000000000000000000000
-    - Coordinator:
-        _application: $TransparentUpgradeableProxy:LynxTACoChildApplication
-        _timeout: 3600
-        _maxDkgSize: 4
-        _admin: $deployer
-        _currency: $LynxRitualToken
-        _feeRatePerSecond: 1
-    - GlobalAllowList:
-        _coordinator: $Coordinator
-        _admin: $deployer
+  - MockPolygonChild
+  - ProxyAdmin
+  - LynxTACoChildApplication:
+      _rootApplication: $MockPolygonChild
+      _minimumAuthorization: 40000000000000000000000
```

Comment:
Something to think about here is that the value used in `root.yml` for the root application would need to match the value used here... 🤔  (cc @KPrasch )

Reply:
That's why we need a separate constants section or file

Reply:
Indeed. Now that it is a yml file, perhaps we can specify a dependent constants.yml file perhaps...I'm less familiar with yml spec.